src/main/java/com/kat/bookstore/controller/shoppingcart/ShoppingCartController.java

```diff
@@ -0,0 +1,77 @@
+package com.kat.bookstore.controller.shoppingcart;
+
+import com.kat.bookstore.dto.cartitem.CartItemDto;
+import com.kat.bookstore.dto.cartitem.CartItemQuantityRequestDto;
+import com.kat.bookstore.dto.cartitem.CreateCartItemRequestDto;
+import com.kat.bookstore.dto.shoppingcart.ShoppingCartDto;
+import com.kat.bookstore.entity.user.User;
+import com.kat.bookstore.service.cartitem.CartItemService;
+import com.kat.bookstore.service.shoppingcart.ShoppingCartService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Positive;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Book Store", description = "Endpoints for managing user's shopping cart")
```

Comment:
Please make the name more precise, for example:
```suggestion
@Tag(name = "Shopping Cart management", description = "Endpoints for managing user's shopping cart")
```